### DISCUSSION TITLE
Will model file (.pt )geting bigger if there are more objects to be trained in my custom model ?

### DISCUSSION THREAD

**Discussion by RioRocker97** [ultralytics/yolov5#1986] [Q&A]
i was wondering what would happen to model file size if i train my custom model to detect 10 objects or more . Should i seperate my custom model for specific object in order to reduce model file size ?

**Comment by glenn-jocher** [comment]
@RioRocker97 you can create a YOLOv5 model with arbitrary class count using PyTorch Hub and it will show you the exact parameter count of your specific model. See PyTorch Hub tutorial for more details: https://github.com/ultralytics/yolov5/issues/36

### Number of Classes
To load a pretrained YOLOv5s model with 10 output classes rather than the default 80:
```python
model = torch.hub.load('ultralytics/yolov5', 'yolov5s', pretrained=True, classes=10)
```
In this case the model will be composed of pretrained weights **except for** the output layers, which are no longer the same shape as the pretrained output layers. The output layers will remain initialized by random weights.
